brainscore_language/plugins/futrell2018/benchmark.py

```diff
@@ -0,0 +1,96 @@
+        super(Futrell2018Pearsonr, self).__init__(
+            identifier='Futrell2018-pearsonr',
+            version=1,
+            parent='behavior',
+            ceiling_func=lambda: ceiler(self.data),
+            bibtex=BIBTEX)
+
+    def __call__(self, candidate: ArtificialSubject) -> Score:
+        candidate.perform_behavioral_task(ArtificialSubject.Task.reading_times)
+        stimuli = self.data['word'].values
+        predictions = candidate.digest_text(stimuli)['behavior'].values
+        targets = self.data.mean('subject')  # compare to "average human"
+        raw_score = self.metric(predictions, targets)
+        score = ceiling_normalize(raw_score, self.ceiling)
+        return score
+
+
+def ceiling_normalize(raw_score, ceiling):
+    # normalize by ceiling, but not above 1
+    score = raw_score / ceiling
+    score.attrs['raw'] = raw_score
+    score.attrs['ceiling'] = ceiling
+    if score > 1:
+        overshoot_value = score.item()
+        # ideally we would just update the value, but I could not figure out how to update a scalar DataArray
+        attrs = score.attrs
+        score = type(score)(1, coords=score.coords, dims=score.dims)
+        score.attrs = attrs
+        score.attrs['overshoot'] = overshoot_value
+    return score
+
+
+benchmarks['Futrell2018-pearsonr'] = Futrell2018Pearsonr
+
+
+class SplitHalvesConsistency:
+    # following
+    # https://github.com/brain-score/brain-score/blob/c51b8aa2c94212a9ac56c06c556afad0bb0a3521/brainscore/metrics/ceiling.py#L25-L96
+
+    def __init__(self, num_splits: int, split_coordinate: str, consistency_metric: Metric):
+        """
+        :param num_splits: how many times to create two halves
+        :param split_coordinate: over which coordinate to split the assembly into halves
+        :param consistency_metric: which metric to use to compute the consistency of two halves
+        """
+        self.num_splits = num_splits
+        self.split_coordinate = split_coordinate
+        self.consistency_metric = consistency_metric
+
+    def __call__(self, assembly: DataAssembly) -> Score:
+        split_dim = np.array(assembly[self.split_coordinate].dims).item()
+        split_values = assembly[self.split_coordinate].values
+        random_state = RandomState(0)
+        consistencies = []
+        splits = range(self.num_splits)
+        for _ in splits:
+            half1_values = random_state.choice(split_values, size=len(split_values) // 2, replace=False)
+            half2_values = set(split_values) - set(half1_values)  # this only works because of `replace=False` above
+            half1 = assembly[{split_dim: [value in half1_values for value in split_values]}].mean(split_dim)
+            half2 = assembly[{split_dim: [value in half2_values for value in split_values]}].mean(split_dim)
+            consistency = self.consistency_metric(half1, half2)
+            # Spearman-Brown correction for sub-sampling
```

Comment:
cool with the correction! if the default is that it is corrected, I think that should go somewhere in the benchmark name?

Reply:
Ah yea the naming is a constant struggle 😄 I think this is one of many many many settings we could put into the identifier (e.g. exact parameters of metric, subject exclusion criteria, number of stimuli, ...), my sense is that just putting `<data>-<metric>` is an okay trade-off between readability and expressiveness. What do you think?

Reply:
oh I see. Is the idea behind non-SB-corrected to reflect the PNAS'21 benchmark? Because this benchmark is already different from the one in the PNAS'21 paper since it asks the model to output reading times rather than for representations that it linearly regresses over.

Reply:
yep. both that, but also just to have metrics that account for SB and ones that do not (in general, for other benchmarks too)

Reply:
I think SB is the correct thing to do for correcting sub-sampling during ceiling estimation, so it's not clear to me why we would want the non-SB benchmark too. If SB is not the correct thing to do, we should *only* have the non-SB benchmark. Can you elaborate why both should be there?

Reply:
I hear you -- mostly for comparison with other work that does not perform it. but I don't feel *super* strongly regarding it, I just like having un-corrected values too

Reply:
ok I think the solution might be `attrs` again in that case 😄

Reply:
that can just be your default reply to my comments from now on :D